lib/global-styles.php

```diff
@@ -278,22 +363,22 @@ function gutenberg_experimental_global_styles_is_site_editor() {
  * @return array New block editor settings
  */
 function gutenberg_experimental_global_styles_settings( $settings ) {
-	if (
-		! gutenberg_experimental_global_styles_has_theme_support() ||
-		! gutenberg_experimental_global_styles_is_site_editor()
-	) {
+	if ( ! gutenberg_experimental_global_styles_has_theme_support() ) {
 		return $settings;
 	}
-
-	$settings['__experimentalGlobalStylesUserEntityId'] = gutenberg_experimental_global_styles_get_user_cpt_id();
+	if ( gutenberg_experimental_global_styles_is_site_editor() ) {
+		$settings['__experimentalGlobalStylesUserEntityId'] = gutenberg_experimental_global_styles_get_user_cpt_id();
+	}
 
 	$global_styles = array_merge(
 		gutenberg_experimental_global_styles_get_core(),
 		gutenberg_experimental_global_styles_get_theme()
 	);
 
 	$settings['__experimentalGlobalStylesBase'] = $global_styles;
```

Comment:
mmm, is `__experimentalGlobalStylesBase` needed by the block editor?

Reply:
It seems it is not needed and it seems we have other settings not used at all. I will open a separate PR to remove unused settings.

Reply:
mmm, how so? `__experimentalGlobalStylesUserEntityId` is used in the site editor to retrieve the CPT through the entities API and `__experimentalGlobalStylesBase` is used in the site editor as well to know which is the base styles the user will overwrite.

Reply:
I think I'm missing something for example when searching for __experimentalGlobalStylesUserEntityId in the edit-site package nothing is found, how does edit-site accesses the setting?

Reply:
It's used by the [GlobalStylesProvider](https://github.com/WordPress/gutenberg/pull/20530/files#diff-641afe1a0152b0e5136c8b46edb84b33R68) but it the branch that introduces that hasn't landed yet.